tests/datavolume_test.go

```diff
@@ -166,6 +170,55 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 			return utils.NewDataVolumeForUpload(dataVolumeName, size)
 		}
 
+		createVddkDataVolume := func(dataVolumeName, size, url string) *cdiv1.DataVolume {
+			// Find vcenter-simulator pod
+			pod, err := utils.FindPodByPrefix(f.K8sClient, f.CdiInstallNs, "vcenter-deployment", "app=vcenter")
+			Expect(err).ToNot(HaveOccurred())
+			Expect(pod).ToNot(BeNil())
+
+			// Get test VM UUID
+			id, err := RunKubectlCommand(f, "exec", "-n", pod.Namespace, pod.Name, "--", "cat", "/tmp/vmid")
+			Expect(err).To(BeNil())
+			vmid, err := uuid.Parse(strings.TrimSpace(id))
+			Expect(err).To(BeNil())
+
+			// Create openshift-cnv namespace
+			ns, err := utils.GetOrCreateNamespace(f.K8sClient, common.VddkConfigMapNamespaces[0])
```

Comment:
Everything should exist in the CDI namespace. see utils.GetNamespace()

Reply:
This is the part of the test that creates the namespace that holds the ConfigMap referencing the vddk-init container image, so it's this explanation again: https://github.com/kubevirt/containerized-data-importer/pull/1320#discussion_r485126128

Reply:
Also replaced with GetNamespace now.

Reply:
I think GetNamespace doesn't work here, it seems to always return "cdi". This works in my local test cluster but not in CI.

Reply:
Yeah, you figured it out already in functional tests its f.CdiInstallNs